W-12374829-disableCloudHubLogs-dm (#2510)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/deploy-to-cloudhub.adoc

include::mule-runtime::partial$mmp-concept.adoc[tag=connectedAppsParameterDescription]
| `applyLatestRuntimePatch` | When set to `true`, the plugin instructs CloudHub to update the worker to the latest available patch for the Mule runtime engine version specified in the deployment configuration, and then deploys the application. +
By default, it is set to `false`. | No

> | `disableCloudHubLogs` | When set to `true`, the plugin instructs CloudHub to disable the CloudHub logging and rely on an application provided by the `log4j2.xml` file. +

Reviewer: ```suggestion
| `disableCloudHubLogs` | When set to `true`, the plugin instructs CloudHub to disable CloudHub logging and instead use the application configured in the `log4j2.xml` file. +
```

Reviewer: (Not sure if that is correct)

> By default, it is set to `false`. | No

Reviewer: "The default value is `false`."?
